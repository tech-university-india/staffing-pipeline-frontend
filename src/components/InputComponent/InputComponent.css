.login-input{
    width: 289.71px;
    height: 36.48px;
    background: #D9D9D9;
}
.search-input{
    box-sizing: border-box;
    width: 538px;
    height: 44px;
    background: #FFFFFF;
    border: 1px solid #051B2C
}