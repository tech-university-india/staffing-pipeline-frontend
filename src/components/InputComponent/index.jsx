import React from 'react';
import './InputComponent.css';
import propTypes from 'prop-types';
export default function InputComponent(props) {
  return (
    <div>
      <input type="text" className="text-input" placeholder={props.placeholder} />
    </div>
  );
}
InputComponent.propTypes = {
  placeholder: propTypes.string,
};
