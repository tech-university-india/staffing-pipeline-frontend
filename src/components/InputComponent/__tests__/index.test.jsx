import React from 'react';
import { render } from '@testing-library/react';
import InputComponent from '..';

describe('InputComponent', () => {
  it('should render correctly', () => {
    const { asFragment } = render(
        <InputComponent placeholder="email" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
