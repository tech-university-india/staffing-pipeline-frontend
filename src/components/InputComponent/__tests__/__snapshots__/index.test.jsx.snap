// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputComponent should render correctly 1`] = `
<DocumentFragment>
  <div>
    <input
      class="text-input"
      placeholder="email"
      type="text"
    />
  </div>
</DocumentFragment>
`;
